xml/apache_os.xml

```diff
@@ -148,6 +148,13 @@ against the following patterns to extract OS information.
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
+  <fingerprint pattern=".*\(Red Hat Enterprise (?:Linux)?\).*">
+    <description>Red Hat Enterprise Linux</description>
```

Comment:
The tests fail because we have a requirement that the value for `description` be unique.  Perhaps change it to `Apache OS: Red Hat Enterprise Linux`.